### DISCUSSION TITLE
The loadmap: push this repo to hono

### DISCUSSION THREAD

**Discussion by roccho-dev** [jcs224/hono_sessions#3] [General]
Hi.
I want to here your plan.

Will you push this repo to hono?
Hono maybe will accept new 3rd party middleware as:

* standard ( in hono repo )
* 3rd party
* custom middleware ( as here is )

**Comment by jcs224** [comment]
Hi there,

I haven't really made a plan on whether to apply to put this in the official Hono repo. I haven't heard any feedback on it, so I can only guess it's working for people?

If it's useful and valuable, I will probably look at what it would take to get it incorporated more officially. It's not quite a 1.0-stable library yet, and there are still a couple things I want to work out before I promote it heavily. However, I don't anticipate many, if any, breaking changes to the user-facing API.

@yusukebe do you have any thoughts? Have you heard any feedback about Hono Sessions?

**Reply by roccho-dev** [reply]
Thanks reply.

I am not from hono, just someone who loves hono and was looking for a SESSION feature.
I have seen 3rd party middleware being adopted by hono.

Please note that this is not a suggestion from hono.

**Reply by jcs224** [reply]
No worries. I use it in my own projects, and I'm confident it could be used in production at this point. There are just some things around types and storage drivers that I want to resolve before declaring 1.0

**Reply by yusukebe** [reply]
Hi.

> @yusukebe do you have any thoughts?

I think it's better to keep managing "Hono Session Middleware" in this repository. A major issue is distributing Deno modules via `deno.land/x`. Since we handle 3rd party middleware in the monorepo, publishing them via `deno.land/x` is challenging because it requires separate repositories for each module.

Recently, I've been questioning if managing the monorepo for 3rd party middleware is the best approach. It might be better if @jcs224 can manage this middleware independently, rather than in our monorepo.

**Reply by yusukebe** [reply]
> Have you heard any feedback about Hono Sessions?

I saw some projects are using this Hono Session Middleware. Great work!

**Comment by jcs224** [comment]
Sounds good. I will continue managing the library independently. Thanks for the feedback!
